src/com/mjcode/model/Piece.java

```diff
@@ -27,16 +35,56 @@ public Piece(Player player, Tile tile, int direction) {
      * @return true if the supplied piece belongs to a player's piece, else false
      */
     public boolean canCapture(Piece other) {
-        return false;
+        return other.getPlayer() != this.player;
     }
-
+    /*
     /**
      * The canMoveTo method determines if a pieces can legally move to a specified tile
      * @param tile The specified tile
      * @return True if piece can legally move to that tile, else false
      */
-    public boolean canMoveTo(Tile tile) {
-        return false;
+    /*public boolean canMoveTo(Tile tile) {
+        // Check if tile is occupied by the opponent's piece
+        if (tile.isOccupied()) {
+            return false;
+        }
+        // If not a King
+        if (!isKinged()) {
+            // If not a King
+            return true;
+        } else {
+            return false;
+
+        }
+    }*/
+    // Test method
+    public boolean canMoveTo(int newRow, int newCol) {
```

Comment:
Add a getter for column and row to the Tile class and change the input parameter for this method back to a Tile.